dspace-api/pom.xml

```diff
@@ -336,16 +338,30 @@
     <dependencies>
 
         <dependency>
-            <groupId>org.hibernate</groupId>
-            <artifactId>hibernate-ehcache</artifactId>
+          <groupId>org.hibernate</groupId>
+          <artifactId>hibernate-core</artifactId>
             <exclusions>
                 <!-- Newer version pulled in via Jersey below -->
                 <exclusion>
                     <groupId>org.javassist</groupId>
                     <artifactId>javassist</artifactId>
                 </exclusion>
+                <!-- Different version provided by mockito below -->
+                <exclusion>
+                    <groupId>net.bytebuddy</groupId>
+                    <artifactId>byte-buddy</artifactId>
+                </exclusion>
```

Comment:
Thanks for tracking this down!  I just removed the exclusion.  It seems that Mockito no longer pulls in `byte-buddy`.

This is why nowadays I'll usually pin down a version in `<dependencyManagement>` rather than use an exclusion.